ui/ducks/ramps/types.ts

```diff
@@ -1,6 +1,6 @@
 export type AggregatorNetwork = {
   active: boolean;
-  chainId: number;
+  chainId: number | string;
```

Comment:
Question: Did you add a `string` for CAIP-2 identifiers? If yes, there's a `CaipChainId` type in `@metamask/utils`